src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Casting.java

```diff
@@ -206,6 +224,7 @@ public boolean removeByOutput(ItemStack output) {
             return false;
         }
 
+        @MethodDescription(description = "groovyscript.wiki.removeByInput", example = @Example("fluid('lava')"))
```

Comment:
`lava` has already been removed, and so this will error. consider either marking this example as a comment, or changing it to  be eg `clay`